docs/content/en/docs/core-components/modules.md

```diff
@@ -198,6 +199,58 @@ The result of the step is stored in the following output variables:
 }
 ```
 
+### Manual capability 
+This capability executes [manual Commands](https://docs.oasis-open.org/cacao/security-playbooks/v2.0/cs01/security-playbooks-v2.0-cs01.html#_Toc152256491) and provides them through the [SOARCA api](/docs/core-components/api-manual).
+
+
+<!-- The manual capability will allow an operator to interact with a playbook. It could allow one to perform a manual step that could not be automated, enter a variable to the playbook execution or a combination of these operations.
+
+The main way to interact with the manual step is through SOARCA's [manual api](/docs/core-components/api-manual). The manual step should provide a timeout SOARCA will by default use a timeout of 10 minutes. If a timeout occurs the step is considered as failed. -->
+
+
+
+
+
+#### Success and failure
+
+The manual step is considered successful if a response is made through the [manual api](/docs/core-components/api-manual). The manual command can specify a timeout but if none is specified SOARCA will use a default timeout of 10 minutes. If a timeout occurs the step is considered as failed.
+
+#### Variables
+
+This module does not define specific variables as input, but it requires one to use out_args if an operator want to provide a response to be used later in the playbook.
+
+#### Example
+
+```json
+{
+    "workflow": {
+        "action--7777c6b6-e275-434e-9e0b-d68f72e691c1": {
+            "type": "action",
+            "agent": "soarca--00010001-1000-1000-a000-000100010001",
+            "targets": ["linux--c7e6af1b-9e5a-4055-adeb-26b97e1c4db7"],
```

Comment:
We might also want to mention better how we interpret agents and targets for a manual step. By CACAO specification, one would argue that the `agent` should be the entity performing the manual act itself, and `target`s the systems on which such act is carried out.

But it makes sense to also see SOARCA as `agent` of the manual action step, because it's implemented per SOARCA interface. Also, irrespective of whatever method to collect the return data is used, the manual step should end with someone pushing the out args values to SOARCA via APIs (safe for 3P connectors I think?).

So yeah overall we might want to explain this interpretation of agents/targets/division of responsibilities in the documentation.